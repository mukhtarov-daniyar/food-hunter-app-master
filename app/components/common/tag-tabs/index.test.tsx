import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import TagTabs from './index';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <TagTabs
        activeId={'activeId'}
        data={[]}
        dataKey={'dataKey'}
        onChange={jest.fn()}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
