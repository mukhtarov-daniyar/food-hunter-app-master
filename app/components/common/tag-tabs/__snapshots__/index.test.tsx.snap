// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTScrollView
  ItemSeparatorComponent={[Function]}
  collapsable={false}
  contentContainerStyle={
    {
      "paddingHorizontal": 38.46153846153846,
      "paddingVertical": 38.46153846153846,
    }
  }
  data={[]}
  getItem={[Function]}
  getItemCount={[Function]}
  horizontal={true}
  keyExtractor={[Function]}
  onContentSizeChange={[Function]}
  onGestureHandlerEvent={[Function]}
  onGestureHandlerStateChange={[Function]}
  onLayout={[Function]}
  onMomentumScrollBegin={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  renderScrollComponent={[Function]}
  scrollEventThrottle={50}
  showsHorizontalScrollIndicator={false}
  stickyHeaderIndices={[]}
  style={
    {
      "flexGrow": 0,
    }
  }
  viewabilityConfigCallbackPairs={[]}
>
  <View />
</RCTScrollView>
`;
